docs/modules/ROOT/pages/multimap-configuration.adoc

```diff
@@ -0,0 +1,52 @@
+= Configuring MultiMaps
+:description: pass:q[Hazelcast MultiMap, also known as IMultiMap, is a specialized map where you can store multiple values under a single key. Just like any other distributed data structure implementation in Hazelcast, MultiMap is distributed and thread-safe. You can create and configure the multimaps using the `MultiMap` Custom Resource.]
+
+{description}
+
+== Configuring the `MultiMap` Resource
+
+Below are the configuration options for the `MultiMap` resource. You can find more detailed information in xref:api-ref.adoc#multimapspec[API Reference] page.
+
+[cols="20%m,80%a"]
```

Comment:
Maybe increase the size of the first column so that the field names don't wrap?

Reply:
I used the same value with the other documents, for now I think we don't need to increase it. I checked the preview, the field names don't wrap. Did you see that they are wrapping somehow?

Reply:
It's probably my screen resolution as the column sizes are proportional so no need to make any updates.